game/templates/game/board.html

```
@@ -319,8 +319,19 @@
             </a>
             <span class="badge" id="modeBadge">PVP</span>
         </div>
+
+        
         
         <div class="auth-buttons">
+            <button
+        type="button"
+        id="themeToggle"
```

HTMLHint (1.9.2)

[error] The id value [ themeToggle ] must be unique.

(id-unique)